TeachingRecordSystem/src/TeachingRecordSystem.Api/V3/Implementation/Dtos/QtsInfo.cs

```diff
@@ -23,36 +24,40 @@ public record QtsInfo
         }
 
         var teacherStatus = await referenceDataCache.GetTeacherStatusByIdAsync(qtsRegistration.dfeta_TeacherStatusId.Id);
-        var statusDescription = GetStatusDescription(teacherStatus);
+        var statusDescription = GetStatusDescription(teacherStatus, qtlsDate);
 
         return new()
         {
             Awarded = awardedDate!.Value,
             CertificateUrl = "/v3/certificates/qts",
             StatusDescription = statusDescription,
+            QtlsHasBeenSet = qtlsHasBeenSet
         };
     }
+    private static string GetStatusDescription(dfeta_teacherstatus teacherStatus, DateTime? qtlsDate = null)
+        => qtlsDate.HasValue
+            ? "Qualified Teacher Learning and Skills status"
+            : teacherStatus.dfeta_Value switch
```

Comment:
This needs to consider the QTLS date and the QTS registration date and return the description for the earliest one.